frontend/src/components/UserProfile.css

```diff
@@ -0,0 +1,81 @@
+*{
+    box-sizing: border-box;
+}
+.div-wrap-nav {
```

Comment:
Usually it's good code style to have a single empty line between each css rule, like 

```
foo {
    ...
}

bar {
    ...
}
```

Reply:
done